### ISSUE TITLE
New OS install, using same SSE

### ISSUE THREAD

**Issue by kleshas** [rockerbacon/lutris-skyrimse-installers#191]
Every couple of months, I reinstall arch.  For the last few months I've had no issues with reinstalling MO2 to lutris to the data drive, overwriting the existing MO2 install.

This time though, things are different.  MO2 runs fine, the profile and the mods all load fine, and are checked (profile and mods folders are on the data partition, not the OS partition).  Loading into SSE, getskseversion shows the latest.  However, loading a saved game gives the "the following mods are not loaded..." and then lists what is probably all the mods checked in MO2.  Loading the game shows what looks like the vanilla experience.  The saved game list in MO2 doesn't have any missing mods (saves tab, right click a saved game to show any missing mods).

I am pretty sure it's something I forgot to do after the OS reinstall, but I'm at a loss atm.  Any help? (is there a better place for this kind of query?)

**Comment by rockerbacon** [comment]
There are reports of VFS not working properly on newer versions of Wine. Check that SSE is set to use Proton 5.0 (not 5.13) in Steam.

**Comment by kleshas** [comment]
That was certainly a step I'd forgotten, but it didn't work.  However, --protonver 5.* is an argument in run.sh.  How does that interact with the setting in steam? (I even tried 4.11 in steam just to check, that didn't work either and then noticed this argument in run.sh and wondered which takes priority)

**Comment by rockerbacon** [comment]
If you're getting `--protonver 5.*` you're using an outdated installer, it should read `--protonver 5.0`. Proton got its version frozen at 5.0 in #162, which was released in version [2.8.5](https://github.com/rockerbacon/lutris-skyrimse-installers/releases/tag/2.8.5) of the installer.

I can't recommend trying to use a newer installer because of the problems being tackled in #182. My recommendation is for you to:
1. Recreate the SSE prefix by deleting `<steam library>/steamapps/compatdata/489830` and launching SSE on Steam. Remember to make sure it is configured to use Proton 5.0
2. Reinstall MO2 using your current installer
3. Manually edit `run.sh` and `download.sh` to use `--protonver 5.0`

**Comment by rockerbacon** [comment]
> I guess run.sh uses its argument then when it runs skse, the steam argument gets used?

`run.sh` launches `$HOME/.local/share/modorganizer2/proton-launcher.sh` which launches Proton directly. The Steam configuration is only to ensure the Wine prefix created by Steam uses the same Wine version that is expected by the installer and the launcher.

**Comment by kleshas** [comment]
> If you're getting `--protonver 5.*` you're using an outdated installer, it should read `--protonver 5.0`. Proton got its version frozen at 5.0 in #162, which was released in version [2.8.5](https://github.com/rockerbacon/lutris-skyrimse-installers/releases/tag/2.8.5) of the installer.

Interesting.  I always download whatever is the latest installer.  Poking around, the lutris cache (on the data drive) is from Aug.  Myabe the installer checks this folder for files, but not versions, and used this for the various install tasks it runs?

**Comment by kleshas** [comment]
> If you're getting `--protonver 5.*` you're using an outdated installer, it should read `--protonver 5.0`. Proton got its version frozen at 5.0 in #162, which was released in version [2.8.5](https://github.com/rockerbacon/lutris-skyrimse-installers/releases/tag/2.8.5) of the installer.
> 
> I can't recommend trying to use a newer installer because of the problems being tackled in #182. My recommendation is for you to:
> 
>     1. Recreate the SSE prefix by deleting `<steam library>/steamapps/compatdata/489830` and launching SSE on Steam. Remember to make sure it is configured to use Proton 5.0
> 
>     2. Reinstall MO2 using your current installer
> 
>     3. Manually edit `run.sh` and `download.sh` to use `--protonver 5.0`

Nothing working.  Always getting the vcrun2019 shasum mismatch, even when i download the one linked in an earlier issue (oddly the installer overwrites that downloaded one in ~/.cache/winetricks/vcrun2019 and replaces with the Sep 28th version) and after winetricks --self-update trick too.

**Comment by kleshas** [comment]
I don't get this.  I noticed the error in my install was the sha256sum mismatch, which caused the installer to download the latest(?) version, which also mismatched.  So I manually downloaded the last 6 versions (which were the only ones available), and tried moving each one to the winetricks cache and installing with each one separately.  None of them worked.  Ignoring the current issues with the latest lutris screwing things up, shouldn't this part of the installer be trivial as it SEEMS to be just simply using the desired vc_redist.x86.exe?
